src/docs/chunkcache.dox

```diff
@@ -0,0 +1,24 @@
+/*! @page chunkcache Chunkcache
+
+@section pinned Pinned Content
+
+This functionality allows users to provide a list of tables that should remain persistently cached, ensuring faster data access times and reducing the need to
+fetch data from potentially much slower block storage. If a user wishes to cache a table in the chunk cache (such that the cached chunks belonging to these tables
+are non-evictable), the pinned config is required.
+
+<b>Note:</b> It is crucial to assess the memory implications of pinning large tables. Ensure the chunkcache has enough capacity to cache the pinned tables.
+
+Example Configuration:
+@code
+chunk_cache=[enabled=true,chunk_size=512KB,capacity=20GB,pinned=("table:chunkcache01", "table:chunkcache02")]
+@endcode
+
+In the config above, the pinned config array contains the names of tables to be pinned. In this example, the tables <b>chunkcache01</b> and <b>chunkcache02</b> are specified.
```

Comment:
NP: Since this is a formal document I would not use abbreviations e.g configuration.

Reply:
Updated in [bd27d2f](https://github.com/wiredtiger/wiredtiger/pull/9613/commits/bd27d2f79f281147659b0885f36e5f06fbe9feba)